v63004/inref/gtm8874.m

```diff
@@ -0,0 +1,209 @@
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;								;
+; Copyright (c) 2018 YottaDB LLC. and/or its subsidiaries.	;
+; All rights reserved.						;
+;								;
+;	This source code contains the intellectual property	;
+;	of its copyright holder(s), and is made available	;
+;	under a license.  If you do not know the terms of	;
+;	the license, please stop and do not read further.	;
+;								;
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;
+testA
+	WRITE "----------",!
+	WRITE "testA",!
+	WRITE "----------",!!
+
+	do checkRegion()
+
+	WRITE !,"VIEW ""STATSHARE"" ",!
+	VIEW "STATSHARE"
+	do checkRegion()
+	WRITE !
+
+	WRITE "STATSHARE DEFAULT: "
+	WRITE $VIEW("STATSHARE","DEFAULT"),!
+	WRITE "STATSHARE AREG:    "
+	WRITE $VIEW("STATSHARE","AREG"),!
+	WRITE "STATSHARE BREG:    "
+	WRITE $VIEW("STATSHARE","BREG"),!
+	WRITE !
+
+	WRITE !,"VIEW ""NOSTATSHARE"" ",!
+	VIEW "NOSTATSHARE"
+	do checkRegion()
+	WRITE !
+
+	WRITE "STATSHARE DEFAULT: "
+	WRITE $VIEW("STATSHARE","DEFAULT"),!
+	WRITE "STATSHARE AREG:    "
+	WRITE $VIEW("STATSHARE","AREG"),!
+	WRITE "STATSHARE BREG:    "
+	WRITE $VIEW("STATSHARE","BREG"),!
+	WRITE !
+
+	WRITE "VIEW ""STATSHARE"":""AREG"""
+	VIEW "STATSHARE":"AREG"
+	WRITE !
+	do checkRegion()
+	WRITE !
+
+	WRITE "STATSHARE DEFAULT: "
+	WRITE $VIEW("STATSHARE","DEFAULT"),!
+	WRITE "STATSHARE AREG:    "
+	WRITE $VIEW("STATSHARE","AREG"),!
+	WRITE "STATSHARE BREG:    "
+	WRITE $VIEW("STATSHARE","BREG"),!
+	WRITE !
+
+	WRITE "VIEW ""STATSHARE"":""DEFAULT""",!
+	VIEW "STATSHARE":"DEFAULT"
+	WRITE !
+	do checkRegion()
+	WRITE !
+	WRITE "VIEW ""NOSTATSHARE"":""AREG""",!
+	VIEW "NOSTATSHARE":"AREG"
+	WRITE !
+	do checkRegion()
+	WRITE !
+
+	WRITE "STATSHARE DEFAULT: "
```

Comment:
dump of the $VIEW(STATSHARE) of all regions and the regionless one all can be moved into checkRegion() function that way we get a dump of open .gst files as well as the $VIEW(STATSHARE) output after each VIEW "STATSHARE" in the test. And that would remove a lot of the $VIEW(STATSHARE) statements in this M program.